src/main/java/core/basesyntax/TransactionProcessor.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.operationstrategy.OperationStrategy;
+import java.util.List;
+
+public class TransactionProcessor {
+    private final OperationStrategy operationStrategy;
+    private final Storage storage;
+
+    public TransactionProcessor(Storage storage) {
+        this.storage = storage;
+        this.operationStrategy = new OperationStrategy();
+    }
```

Comment:
Storage has the static method, use as `Storage. fruits()`
```suggestion
    public TransactionProcessor(OperationStrategy operationStrategy) {
        this.operationStrategy = operationStrategy;
    }
```